bitnami/grafana-operator/Chart.yaml

```diff
@@ -0,0 +1,40 @@
+apiVersion: v2
+name: grafana-operator
+description: A Helm chart for Kubernetes
```

Comment:
This can be replaced by something more descriptive, e.g:

"Kubernetes Operator based on the Operator SDK for creating and managing Grafana instances"

What do you think?